src/app/work-item/work-item-detail/work-item-detail.component.html

```diff
@@ -10,72 +10,84 @@
+                        [(ngModel)]="workItem.fields['system.description']"
+                        name="description"></textarea>
+                    </div>
                 </div>
-            </div>
-            <div class="form-group">
-                <label class="col-sm-1 control-label" for="wi-detail-desc">Description</label>
-                <div class="col-sm-5">
-                <textarea id="wi-detail-desc" placeholder="Description" class="form-control"
-                    [(ngModel)]="workItem.fields['system.description']"
-                    name="description"
-                ></textarea>
+                <div class="form-group">
+                    <label class="col-sm-1 control-label" for="wi-detail-type">
+                        Type
+                    </label>
+                    <div class="col-sm-5">              
+                        <select id="wi-detail-type" class="form-control" 
+                            [(ngModel)]="workItem.type"
+                            name="type" #type="ngModel">
+                                <option *ngFor="let type of workItemTypes" 
+                                [value]="type.name"
+                                [selected]='type.name==workItem.type'>
+                                {{type.name.substring(7,type.length)}}                  
+                                </option>
+                        </select>      
+                    </div>
                 </div>
-            </div>
-            <div class="form-group">
-                <label class="col-sm-1 control-label" for="wi-detail-type">Type</label>
-                <div class="col-sm-5">
-                <select id="wi-detail-type" class="form-control" 
-                    [(ngModel)]="workItem.type"
-                    name="type" #type="ngModel">
-                    <option *ngFor="let type of workItemTypes" [value]="type">{{type.substring(7,type.length)}}</option>
-                </select>          
+                <div class="form-group">
+                    <label class="col-sm-1 control-label" for="wi-detail-creator">Creator</label>
+                    <div class="col-sm-5">
+                    <input type="text" id="wi-detail-creator" placeholder="Creator" class="form-control"
+                        [(ngModel)]="workItem.fields['system.creator']"
+                        name="creator" />
+                    </div>
                 </div>
-            </div>
-            <div class="form-group">
-                <label class="col-sm-1 control-label" for="wi-detail-creator">Creator</label>
-                <div class="col-sm-5">
-                <input type="text" id="wi-detail-creator" placeholder="Creator" class="form-control"
-                    [(ngModel)]="workItem.fields['system.creator']"
-                    name="creator" />
+                <div class="form-group">
+                    <label class="col-sm-1 control-label" for="wi-detail-assignee">Assignee</label>
```

Comment:
@joshuawilson you are right. Not sure why I changed that. I will revert the change.